### PR TITLE
README.md in Portuguese PT-PT

### PR DESCRIPTION
**Pull request by LiTO773**
Here's the README.md in Portguese (Portugal). Hope you guys like it :)

Feel free to make any changes

### CODE REVIEW

**Comment by bitcrazed** [comment]
I think PR #71 and #78 already translate to PT-BR, no?

**Comment by LiTO773** [comment]
PT-PT's grammar is a bit different from PT-BR

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
-# License
-All files within this repo are released under the [MIT (OSI) License]( https://en.wikipedia.org/wiki/MIT_License) as per the [LICENSE file](https://github.com/Microsoft/MS-DOS/blob/master/LICENSE.md) stored in the root of this repo.
+# Licença
+Todos os ficheiros deste repositório estão lançados sob a [Licença MIT (OSI)](https://pt.wikipedia.org/wiki/Licen%C3%A7a_MIT) como descrito no [ficheiro LICENSE](https://github.com/Microsoft/MS-DOS/blob/master/LICENSE.md) presente na raiz deste repositório.
 
-# Contribute!
-The source files in this repo are for historical reference and will be kept static, so please don’t send Pull Requests suggesting any modifications to the source files, but feel free to fork this repo and experiment 😊.  
+# Contribua!
+Os ficheiros de código fonte deste repositório são para referência histórica, logo não serão alterados. Deste modo, pedimos que não envie Pull Requests a sugerir quaisquer tipo de alterações a estes ficheiros, mas sinta-se à vontade de fazer um fork deste repositório e fazer alterações 😊.
```
Comment (pedrocatalao) [inline README.md]: I'd use "não hesite" instead of "sinta-se à vontade" since it's a more natural way of putting it in PT-PT

**Comment by bitcrazed** [comment]
Ok. But can't replace English text in English readme. Please move translation into a new file with the appropriate suffix (see other translations for examples).

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ These are the same files [originally shared at the Computer History Museum on Ma
 All files within this repo are released under the [MIT (OSI) License]( https://en.wikipedia.org/wiki/MIT_License) as per the [LICENSE file](https://github.com/Microsoft/MS-DOS/blob/master/LICENSE.md) stored in the root of this repo.
 
 # Contribute!
-The source files in this repo are for historical reference and will be kept static, so please don’t send Pull Requests suggesting any modifications to the source files, but feel free to fork this repo and experiment 😊.  
+The source files in this repo are for historical reference and will be kept static, so please **don’t send** Pull Requests suggesting any modifications to the source files, but feel free to fork this repo and experiment 😊.
```
[the remark this answers is not in the document]
Comment (LiTO773) [reply README.md]: Really sorry about that, I didn't notice it at the time :/

**Comment by bitcrazed** [comment]
Thanks all. Merged.

**Pull request merged**